tests/cypress/integration/2-general-onboarding-flow/get-started-experience.cy.js

```diff
@@ -65,6 +65,45 @@ describe( 'Start Setup WP Experience Page', function () {
 		} );
 	} );
 
+	const EventsAPI = ( experience_val ) => {
+		cy.intercept('http://localhost:8882/index.php?rest_route=%2Fnewfold-onboarding%2Fv1%2Fevents%2Fbatch&flow=wp-setup&_locale=user').as('events');
+		cy.wait('@events').then((interception) => {
+			const responseBody = interception.request.body;
+			const responseData1 = responseBody[0].data;
+			const responseData2 = responseBody[1].data;
+			if("experience_level" in responseData1){
+				expect(responseData1.experience_level).equal(experience_val);
+			};
+			if("experience_level" in responseData2){
+				expect(responseData2.experience_level).equal(experience_val);
+			};
+		});
+	};
+
+	it( 'Check if events API call being made after radio buttons are clicked', () => {
+		let radioCount = 0;
+		const className = '.components-radio-control__option';
+		const arr = cy.get( className );
+		arr.each( () => {
+			cy.reload();
```

Comment:
Do we need to `reload` the page every time we click a radio button? That's not in line with how the user will be clicking the buttons.

Reply:
Yes, this'll be needed to clear the older network logs as every time when a radio button is clicked, the API call is overlapped on the earlier one (happening only on cypress). So, to eliminate this, we have to use reload before every radio button click.

Reply:
Removed reload.